libcloud/compute/drivers/upcloud.py

```diff
@@ -263,11 +270,13 @@ def _parse_country(self, zone_id):
         Zone_id format [country]_[city][number], like fi_hel1"""
         return zone_id.split('-')[0].upper()
 
-    def _to_node_sizes(self, plans):
-        return [self._construct_node_size(plan) for plan in plans]
+    def _to_node_sizes(self, plans, prices):
+        plan_price = PlanPrice(prices)
+        return [self._construct_node_size(plan, plan_price) for plan in plans]
 
```

Comment:
Curious about the naming, here. Why `_construct_node_size` instead of the more common (across Libcloud compute drivers) `_to_node_size`? I know it's an internal method, and this name was already in use in prior commits, but it still strikes me as unusual.